File: python/BRIDGE_WHOLE_PART/BRIDGE_WHOLE_PART/whole_part/membresia/MembresiaSencilla.py
from BRIDGE_WHOLE_PART.bridge.ImplAbstractaMembresia import ImplAbstractaMembresia

class MembresiaSencilla(ImplAbstractaMembresia):
    
    def __init__(self):
        pass

    def completarMembresia(self, datos):
        super().completarMembresia(datos)
    
    def mostrarMembresia(self):
        print(
        f'''
        Reservación:
        
        Datos cliente:
            Nombres          : {self.datosCliente.nombres}
            Apellido paterno : {self.datosCliente.apellidoPaterno}
            Apellido materno : {self.datosCliente.apellidoMaterno}
            Edad             : {self.datosCliente.edad}
            Tipo de cliente  : {self.datosCliente.tipoCliente}
        '''
        )

File: php/BRIDGE_WHOLE_PART/whole_part/membresia/MembresiaSencilla.php
<?php
require_once './bridge/ImplAbstractaMembresia.php';

class MembresiaSencilla extends ImplAbstractaMembresia{
    
    public function __construct() {
        parent::__construct();
    }
    
    public function completarMembreisa($datos) {
        parent::completarMembreisa($datos);
    }
    
    public function mostrarMembresia() {
        return parent::mostrarMembresia();
    }

}
